src/resources/users/index.tsx

```diff
@@ -18,9 +20,23 @@ const UserCreate = (): React.ReactElement => {
 
 const UserEdit = (): React.ReactElement => {
   const path: string = `/${constants.R_USERS}`
+  const audit = useAudit()
 
   return (
     <Edit
+      mutationOptions={{
```

Comment:
Thanks @TahaKhanAbdalli - but I could with detail of what got edited.  We can't show the password, but it would be useful to know if the role changed, username got edited, etc.   Since there are so few editable properties on the form, I guess you don't have to use the complex processing like we do in `Edit Item` - maybe just do a manual comparison, and provide a suitable description for the old values.